web/frontend/vite.config.ts

```diff
@@ -28,11 +30,14 @@ export default defineConfig({
       output: {
         entryFileNames: '[name].js',
         chunkFileNames: (chunk) => {
+          // @ts-expect-error Fix this
```

Comment:
Is the fix something like "use `node:crypto` instead of `jshashes`" or "don't bother customizing this ourselves"?

Reply:
Yeah this vite config, especially the output section, is very stinky and needs a redo, wanted to leave it outside the scope of this

Reply:
I don't remember why - @DTCurrie might - we needed to do manual filename hashing. I bet it was for a reason that might not exist anymore. But yeah, even if we do we could use node:crypto or `nanoid`